### PR TITLE
Tianwei/doc tutorial

### PR DESCRIPTION
**Pull request by twni2016**
Fixed mkdocs installation issue and change the experiment output folder name for gitignore.

### CODE REVIEW

File: conda/environment-dev.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ dependencies:
   - markdown=3.3
   - mkdocs=1.1.2
   - mkdocs-material=5.5.3
+  - mkdocs-material-extensions>=1.0
```
Comment (jordis-ai2) [inline conda/environment-dev.yml:11]: Could you tell us how this was an issue (i.e. how you reached the need for this dependency)? I don't recall having to explicitly add this before.

**Review by jordis-ai2 — COMMENTED** [review]
I'm very happy with the enhancement for the first experiment tutorial, but I would like to double check the new dependency is actually missing and not an artifact derived from a local setup.

File: conda/environment-dev.yml (hunk 1 above)
Comment (twni2016) [reply conda/environment-dev.yml:11]: Sure. After I installed the full library by conda, when I ran `mkdocs serve`, I got an error in mkdocs:
```
...
  File "/net/nfs2.prior/tianwein/miniconda3/envs/allenact/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2472, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/net/nfs2.prior/tianwein/miniconda3/envs/allenact/lib/python3.6/site-packages/pkg_resources/__init__.py", line 772, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'mkdocs-material-extensions>=1.0' distribution was not found and is required by the application
```
Then I install the `mkdocs-material-extensions` to solve this issue.

Comment (jordis-ai2) [reply conda/environment-dev.yml:11]: Ok, thanks

**Review by jordis-ai2 — APPROVED** [review]
LGTM

**Review by Lucaweihs — APPROVED** [review]
LGTM.

**Pull request merged**